BEP188.md

```diff
@@ -0,0 +1,24 @@
+# BEP-?: Early Broadcast block for an in-turn validator.
+
+## 1. Summary 
+
+This BEP introduces a new block broadcast strategy to improve the network capacity and stability. In-turn validators can broadcast its block earlier when it is ready, so that other nodes could import the block earlier and have more time to mine or import the next block.
+
+## 2. Abstract 
+
+Currently, a mined block will not be broadcasted until the timestamp of header.Time is reached. This proposal tries to allow an in-turn validator to broadcast earlier when the block is ready. But if all the validators broadcast blocks earlier without any restrictions, the blockchain will grow too fast. To solve this problem, we also limit broadcast time to be after the parent block’s header.Time.
+
+## 3. Status
```

Comment:
pls follow the latest BEP format, defined in [BEP1.md](https://github.com/bnb-chain/BEPs/blob/master/BEP1.md#5--bep-format)